crates/rpc/src/lib.rs

```diff
@@ -43,8 +44,12 @@ impl RpcServer {
 
     /// Starts the HTTP-RPC server.
     pub async fn run(self) -> Result<(ServerHandle, SocketAddr), anyhow::Error> {
+        const TEN_MB: u32 = 10 * 1024 * 1024;
```

Comment:
Was this not by default 1 MB?

Reply:
I was surprised too :smile: : https://github.com/eqlabs/jsonrpsee/blob/master/server/src/server.rs#L229